templates/silicon/index.html

```diff
@@ -0,0 +1,550 @@
+          alt="build your own stack and prototype your application illustration",
+          width="600",
+          height="336",
+          hi_def=True,
+          loading="lazy"
+          ) | safe
+        }}
+      </div>
+      <p>
+        Ubuntu on workstations, servers, IOT devices, edge devices allows for seamless developer experience thus reducing cost and friction for application development and maintenance.
+      </p>
+      <p>
+        Pro-active pre GA work on next-generation software and silicon hardware through optimised Ubuntu offering ensures silicon features reach both internal and external users in the fastest way possible. Strong Ubuntu community engagement ensures constant evolution of hardware and software features.
+      </p>
+      <p>
+        Explore how the joint solutions offered by Canonical and our silicon partners accelerate the customers&rsaquo; project.
+      </p>
+      <ul class="p-list--divided-bulleted ">
+        <li class="p-list__item">
+          <a href="https://ubuntu.com/download/iot/intel-iotg">
+            Download Ubuntu for Intel IoT platforms
+          </a>
+        </li>
+        <li class="p-list__item">
+          <a href="https://ubuntu.com/download/amd-xilinx">
+            Install Ubuntu on AMD-Xilinx Zynq UltraScale+MPSoC-based evaluation boards and Kria SOMs
+          </a>
+        </li>
+        <li class="p-list__item">
+          <a href="https://ubuntu.com/download/risc-v">
+            Download Ubuntu for RISC-V Platforms
+          </a>
+        </li>
+      </ul>
+    </div>
+  </div>
+</section>
+<section class="p-strip is-shallow">
+  <div class="row">
+    <hr />
+    <div class="col-8 u-sv3">
+      <h2 class="u-sv3">Expand opportunities through joint marketing and co-selling with Canonical</h2>
+    </div>
+    <div class="col-9 col-start-large-4">
+      <div class="u-sv3">
+        <hr />
+      </div>
+      <div class="row">
+        <div class="col-3 u-hide--medium u-hide--small">
+          {{ image (
+            url="https://assets.ubuntu.com/v1/f986d2bf-co-selling-handshake-dark-suru.png",
+            alt="handshake illustration with dark suru backgroud",
+            width="400",
+            height="668",
+            hi_def=True,
+            loading="lazy"
+            ) | safe
+          }}
+        </div>
+        <div class="col-6">
+          <p class="u-sv3 u-no-padding--top">
+            Accelerate design wins by developing joint cost effective solutions - Canonical SW portfolio on latest silicon reduces TCO for end customers.Joint marketing increases silicon reach across ever growing Ubuntu and other open source communities. Take advantage of a clear established route for partners(ODM, OEM, etc) to enable, certify, optimise, promote and sell silicon with Canonical SW. Expand into new domains by leveraging and enhancing projects that Canonical has already successfully delivered for its enterprise and cloud customers.
```

Comment:
```suggestion
            Accelerate design wins by developing joint cost effective solutions &mdash; Canonical SW portfolio on latest silicon reduces TCO for end customers. Joint marketing increases silicon reach across ever growing Ubuntu and other open source communities. Take advantage of a clear established route for partners (ODM, OEM, etc) to enable, certify, optimise, promote and sell silicon with Canonical SW. Expand into new domains by leveraging and enhancing projects that Canonical has already successfully delivered for it's enterprise and cloud customers.
```
We true to use web safe formats for long dashes such as this. It's hard to know which to use, so I usually use this as my reference: https://www.scribbr.com/language-rules/dashes/#:~:text=The%20en%20dash%20is%20approximately,a%20break%20in%20a%20sentence.
Same as above a space is needed here as 'ODM' etc are properties of partner and not part of the name.

Reply:
Thanks Pete, I'll check it out!